package.json

```diff
@@ -37,7 +37,7 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
     "raw-loader": "^0.5.1",
-    "react": "^0.14.8",
+    "react": ">=0.14.8 <15.2.0",
```

Comment:
Done :)

On Wed, Jul 27, 2016 at 10:06 PM, Alan Clarke <notifications@github.com>
wrote:

> In package.json
> <https://github.com/alanclarke/time-input/pull/12#discussion_r72523750>:
>
> > @@ -37,7 +37,7 @@
> >      "karma-webpack": "^1.7.0",
> >      "mocha": "^2.4.5",
> >      "raw-loader": "^0.5.1",
> > -    "react": "^0.14.8",
> > +    "react": ">=0.14.8 <15.2.0",
>
> Btw you can amend the commit? Don't need to create a whole new pr!
> … <#m_3824928818871531861_>
> Sent from my iPhone
> On 27 Jul 2016, at 21:53, Develliot ***@***.***> wrote: In package.json: >
> @@ -37,7 +37,7 @@ > "karma-webpack": "^1.7.0", > "mocha": "^2.4.5", >
> "raw-loader": "^0.5.1", > - "react": "^0.14.8", > + "react": ">=0.14.8
> <15.2.0", No worries I only did this because I couldn't guarantee
> compatibility after 15.1 I'll make a new pull request. … — You are
> receiving this because you commented. Reply to this email directly, view it
> on GitHub, or mute the thread.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/alanclarke/time-input/pull/12/files/ffb249310cf3000a9cad2706191751afe600b091#r72523750>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFGXfgl59vOT13lSizK-vZkHWYH6FFu3ks5qZ8hTgaJpZM4JWBfU>
> .
>



-- 
[image: “audioBoom]*Elliot Revan | Front End Web Developer*
The world’s leading spoken-word audio platform
0207 403 6688 | +44 (0)7958 490 870
@audioBoom <https://www.twitter.com/audioBoom> | audioBoom.com
<https://www.audioboom.com/>
Follow me on *audioBoom* <http://www.audioboom.com/>!